modules/exploits/linux/http/goautodial_3_rce_command_injection.rb

```diff
@@ -42,100 +50,114 @@ def initialize(info={})
 
 
   def check
```

Comment:
The logic I had for this was to check for the software patch version that was evident within the changelog.txt. If the changelog didn't contain the version number (that included the auth bypass fix) then it was presumed to be vulnerable.